packages/protocol/lib/compatibility/ast-code.ts

```diff
@@ -251,12 +251,19 @@ function generateASTCompatibilityReport(oldContract: ZContract, oldArtifacts: Bu
  */
 export function reportASTIncompatibilities(
   oldArtifacts: BuildArtifacts,
-  newArtifacts: BuildArtifacts): ASTCodeCompatibilityReport {
-  const reports = newArtifacts.listArtifacts()
-  .map((newArtifact) => {
-    const oldArtifact = oldArtifacts.getArtifactByName(newArtifact.contractName)
-    const oldZContract = oldArtifact ? makeZContract(oldArtifact) : null
-    return generateASTCompatibilityReport(oldZContract, oldArtifacts, makeZContract(newArtifact), newArtifacts)
-  })
-  return mergeReports(reports)
+  newArtifactsSets: BuildArtifacts[]): ASTCodeCompatibilityReport {
```

Comment:
In future releases, will `oldArtifacts` also be sets of artifacts? Should this code account for that?

Reply:
Will open an issue for this